### PR TITLE
moved spanning multi lines message to appear on some lane vs new line…

### LINKED ISSUE #3620: "Spanning multiple lines" should not be on its own line to reduce confusion.

### PR DESCRIPTION
**Pull request by AnthonyBroadCrawford**
Fixes #3620.  I moved the message to be on the same line per the direction in the issue.  

I didn't add any tests for this, but open to that if we think that is needed.  

Let me know what feedback exists and what you would like to see changed. 

Cheers.

### CODE REVIEW

File: libsolidity/interface/SourceReferenceFormatter.cpp [hunk 1]
```diff
@@ -79,8 +79,7 @@ void SourceReferenceFormatter::printSourceLocation(SourceLocation const* _locati
 			scanner.lineAtPosition(_location->start) <<
 			endl <<
 			string(startColumn, ' ') <<
-			"^\n" <<
```
Comment (axic) [inline libsolidity/interface/SourceReferenceFormatter.cpp:82]: Weird we haven't used `endl` here :)

**Comment by axic** [comment]
This should change from:
```
strings.sol:636:5: Warning: This declaration shadows an existing declaration.
    function rsplit(slice self, slice needle) internal returns (slice token) {
    ^
Spanning multiple lines.
```

to:
```
strings.sol:636:5: Warning: This declaration shadows an existing declaration.
    function rsplit(slice self, slice needle) internal returns (slice token) {
    ^ Spanning multiple lines.
```

**Comment by axic** [comment]
I'd perhaps go one step further and clarify the text somewhat, like "Source spanning multiple lines" or "Referenced source spanning multiple times" ?

File: libsolidity/interface/SourceReferenceFormatter.cpp (hunk 1 above)
Comment (chriseth) [reply libsolidity/interface/SourceReferenceFormatter.cpp:82]: As far as I know, `endl` is exactly `\n` plus "flush the stream", so in most cases, `\n` would actually be better.

**Comment by chriseth** [comment]
```
strings.sol:636:5: Warning: This declaration shadows an existing declaration.
    function rsplit(slice self, slice needle) internal returns (slice token) {
    ^ (Relevant source part starts here and spans across multiple lines).
```

File: libsolidity/interface/SourceReferenceFormatter.cpp (hunk 1 above)
Comment (axic) [reply libsolidity/interface/SourceReferenceFormatter.cpp:82]: Yep, but we consistently use `endl` :)

**Comment by axic** [comment]
@AnthonyBroadCrawford can you please update the text to what @chriseth has suggested  and include an entry in the changelog?

**Comment by AnthonyBroadCrawford** [comment]
@axic On it today.  Had some family time intervene with my availability yesterday. 

/cc @chriseth

File: libsolidity/interface/SourceReferenceFormatter.cpp (hunk 1 above)
Comment (AnthonyBroadCrawford) [reply libsolidity/interface/SourceReferenceFormatter.cpp:82]: @axix @chriseth 

I updated to use endl in this part of the file.  I did not search through the rest.  You can see this reflected in second commit to the PR.

**Comment by AnthonyBroadCrawford** [comment]
@axic @chriseth I updated the language and committed to my branch and updated change log.

**Review by chriseth — APPROVED** [review]

**Comment by chriseth** [comment]
Can you squash the commits into a single one?

**Comment by AnthonyBroadCrawford** [comment]
@chriseth yup!

**Comment by AnthonyBroadCrawford** [comment]
@chriseth squashed and now in a single commit

**Comment by chriseth** [comment]
Thanks!

**Pull request merged**